compiler/rustc_mir/src/transform/coverage/spans.rs

```diff
@@ -656,7 +769,7 @@ impl<'a, 'tcx> CoverageSpans<'a, 'tcx> {
 pub(super) fn filtered_statement_span(
     statement: &'a Statement<'tcx>,
     body_span: Span,
-) -> Option<Span> {
```

Comment:
can you add a comment about what each span is? also it seems like here and below, the second one is just the `body_span` you called it with. is this for convenience elsewhere?

Reply:
Comments added.

The second one is not the `body_span` (sorry if this is not clear).